awx/ui_next/src/screens/Inventory/InventoryList/InventoryListItem.jsx

```diff
@@ -29,6 +30,18 @@ const DataListAction = styled(_DataListAction)`
   grid-template-columns: repeat(2, 40px);
 `;
 
+const Badge = styled(PFBadge)`
+  margin-left: 8px;
+`;
+
+const ListGroup = styled.span`
+  margin-left: 24px;
+
+  &:first-of-type {
```

Comment:
This bit does not seem to be doing anything and I think we can remove it

Reply:
Thank you, removed that part.